selfdrive/camerad/cameras/camera_common.cc

```diff
@@ -0,0 +1,347 @@
+#include <thread>
+#include <stdio.h>
+#include <signal.h>
+
+#if defined(QCOM) && !defined(QCOM_REPLAY)
+#include "cameras/camera_qcom.h"
+#elif QCOM2
+#include "cameras/camera_qcom2.h"
+#elif WEBCAM
+#include "cameras/camera_webcam.h"
+#else
+#include "cameras/camera_frame_stream.h"
+#endif
+
+#include "camera_common.h"
+#include <czmq.h>
```

Comment:
why is zmq needed?

Reply:
only for tici debugging and fl I think? may move to qcom2.cc

Reply:
oh nvm, qcom also uses zmq to do exposure/focus and stuff